src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -9971,6 +9971,22 @@ public async Task RecordStruct()
 ",
                 @"
 record  struct  R(int X);
+");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        public async Task FileScopedNamespace()
+        {
+            await AssertFormatAsync(
+                @"
+namespace NS;
+
+class C { }
+",
+                @"
+namespace NS;
+
+    class C { }
```

Comment:
sweet :)  i think we should try to fix this as part of hte initial PR.  otherwise the feature will feel totally broken.  Up to you if you want to try that.  otherwise I (or other IDE member) can do this :)

Reply:
I think this test might be misleading. The first argument is the `expected` result of formatting, and the second argument is input `code` to the formatter. It looks like from this and from ad-hoc testing in the IDE, we seem to get the result we want from the formatter.

Reply:
oh right.  i hate tehse tests ofr that reason.  i just ran into that with chuck.  :(  no problem then!

Reply:
Added argument labels for clarity.